api/src/main/resources/workbench.yaml

```diff
@@ -1979,6 +2222,130 @@ definitions:
         format: int64
         description: the count for this criteria type in the cohort
 
+  CreateCohortAnnotationDefinitionRequest:
+    type: object
+    required:
+      - name
+      - annotationType
+    properties:
+      name:
+        type: string
+        description: the name of this annotation.
+      annotationType:
+        $ref: "#/definitions/AnnotationType"
+
+  AnnotationType:
+    type: string
+    description: possible values representing the type of annotation.
+    enum: [STRING, BOOLEAN, INTEGER, DATE, TIMESTAMP]
+
+  CohortAnnotationDefinition:
+    type: object
+    required:
+      - cohortAnnotationDefinitionId
+      - cohortId
+      - name
+      - annotationType
+    properties:
+      cohortAnnotationDefinitionId:
+        type: integer
+        format: int64
+        description: the annotation definition id.
+      cohortId:
+        type: integer
+        format: int64
+        description: the cohort id.
+      name:
+        type: string
+        description: the name of this annotation.
+      annotationType:
+        $ref: "#/definitions/AnnotationType"
+      participantCohortAnnotation:
```

Comment:
What is this field for?

Reply:
This is the child ParticipantCohortAnnotation that belongs to the parent CohortAnnotationDefinition. So from a UI perspective this is useful to pair them per participant id. So i think what your getting at is... we should have 2 possibly different api calls.. 1) for CohortAnnotationDefinition and 2) for ParticipantCohortAnnotation. CohortAnnotationDefinition wouldn't be tied to the cdr version/review and ParticipantCohortAnnotations would be tied to the cdr version/review??? I'm trying to couple the 2 but decoupling them works as well.